### ISSUE TITLE
Error 603: "you can't do that right now" when adding team members

### ISSUE THREAD

**Issue by ggraubins** [keybase/client#24191]
I just started receiving error 603 when trying to add members to my teams last night. I just sent a feedback item that includes my logs through the Keybase GUI. It didn't give a "log id" unfortunately, but I think you should be able to find it according to the time? I literally sent it in the past 10 minutes.

Could someone reset my account so it's not considered a spammer? These are members that we both follow each other, are members on my other teams, etc. etc.

Any additional information like my user id I can provide ASAP via a direct message if need be.

Thanks!

**Comment by yoza111** [comment]
I am having the same problem. did you manage to get this fixed?

**Comment by ChicagoGregg** [comment]
Yes, there is a throttle when adding a lot of users to your teams for an anti-spam measure. Mine was fixed. I don't recall how long it took, but I seem to recall about 2 days.

**Comment by yoza111** [comment]
Thanks for your reply because it made me realize I think I was posting my Issue in the wrong place on GitHub. I have retried under Keybase/client and sent through the same feedback report with logs included and explained it the same way you did. Im not very good at technology im only here to get my account fixed so seeing an example really helps. Thumbs up!

**Comment by jflot** [comment]
I have been having the same issue for over two weeks now (https://github.com/keybase/client/issues/24731). I sent my logs three times already (latest was with log ID 012ffd2c66f42f915ece331c), to no avail. This is hugely hindering my work... Any idea what else I could try?
